.circleci/config.yml

```diff
@@ -128,7 +128,18 @@ jobs:
             cd mattermost-server
             make store-layers
             if [[ -n $(git status --porcelain) ]]; then echo "Please update the store layers using make store-layers"; exit 1; fi
-
+  check-mocks:
+    docker:
+      - image: mattermost/mattermost-build-server:20200731_golang-1.14.6
+    working_directory: ~/mattermost
+    steps:
+      - attach_workspace:
+          at: ~/mattermost/
+      - run:
+          command: |
+            cd mattermost-server
+            make einterfaces-mocks store-mocks
+            if [[ -n $(git status --porcelain) ]]; then echo "Please update the mocks using make make einterfaces-mocks store-mocks"; exit 1; fi
```

Comment:
> using make make einterfaces-mocks store-mocks

Is make make correct or just typo? If yes, maybe some single quotation could make it clearer?

Reply:
Looks good to me otherwise! Thank you so much!

Reply:
typo fixed!